docs/pages/deploy/build-project.mdx

```diff
@@ -1,28 +1,124 @@
 ---
 title: Build your project for app stores
-description: Learn how to create production builds of your project that are ready for the app stores.
-sidebar_title: Build for app stores
+description: Learn how to create a production build for your app that is ready to be submitted to app stores from the command line using EAS Build.
+sidebar_title: Build project for app stores
 ---
 
+import { BookOpen02Icon } from '@expo/styleguide-icons/outline/BookOpen02Icon';
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { Collapsible } from '~/ui/components/Collapsible';
 import { Terminal } from '~/ui/components/Snippet';
+import { Tabs, Tab } from '~/ui/components/Tabs';
+
+Whether you have built a native app binary using [EAS](/build/setup/) or [locally](/guides/local-app-development/), the next step in your app development journey is to submit your app to the stores. To do so, you need to create a **production build**.
+
+Production builds are submitted to app stores for release to the general public or as part of a store-facilitated testing process such as TestFlight. This guide explains how to create production builds with [EAS](#production-builds-using-eas) and [locally](#production-builds-locally). It is also possible to create production builds for Expo apps with any CI service capable of compiling Android and iOS apps.
+
+## Production builds using EAS
+
+Production builds must be installed through their respective app stores. They cannot be installed directly on your Android Emulator or device, or iOS Simulator or device. The only exception to this is if you explicitly set `"buildType": "apk"` for Android on your build profile. However, it is recommended to use **aab** when submitting to stores, and this is the default configuration.
+
+### `eas.json` configuration
+
+A minimal configuration for building a production build in **eas.json** that is already created when you create your first build:
+
+{/* prettier-ignore */}
+```json eas.json
+{
+  "build": {
+    /* @hide ... */ /* @end */
+    "production": {}
+    /* @hide ... */ /* @end */
+  }
+}
+```
+
+### Create a production build
+
+To create a production build, run the following command for a platform:
+
+<Tabs tabs={['Android', 'iOS']}>
+  <Tab>
+    <Terminal cmd={['$ eas build --platform android']} />
+  </Tab>
+  <Tab>
+    <Terminal cmd={['$ eas build --platform ios']} />
+  </Tab>
+</Tabs>
+
+> You can attach a message to the build by passing `--message` to the build command, for example, `eas build --platform ios --message "Some message"`. The message will appear on the website. It comes in handy when you want to leave a note with the purpose of the build for your team.
```

Comment:
```suggestion
> You can attach a message to the build by passing `--message` to the build command, for example, `eas build --platform ios --message "Some message"`. The message will appear on the Expo dashboard. It comes in handy when you want to specify the purpose of the build for your team.
```

- In the above, the "website" is vague. Let's replace it with the Expo dashboard.
- Let's remove the wordiness in the last sentence.